lib/juttle-viewer-view.coffee

```diff
@@ -0,0 +1,43 @@
+{ScrollView} = require 'atom-space-pen-views'
+JuttleClient = require 'juttle-client-library'
+
+module.exports =
+class JuttleViewerView extends ScrollView
+  @content: ->
+    @div class: 'juttle-viewer native-key-bindings', tabindex: -1
+
+  constructor: (@editorId) ->
+    super
+    juttleClient = new JuttleClient(atom.config.get('juttle-viewer.juttleServiceHost'))
+    @juttleClientView = new juttleClient.View(@element);
+
+    @juttleClientView.on 'error', (err) =>
+      @_showError(err)
+
+    @juttleClientView.on 'warning', (warn) =>
+      @_showWarning(warn)
```

Comment:
actually opening mulitple juttles, but this should have a deconstructor when they close `juttle-viewer`